dist/index.js

```diff
@@ -8664,7 +8662,7 @@ module.exports.postUpdateComment = async function (appName, appUrl) {
   const comment = formatComment({
     image: owlberts.update,
     imageLink: appUrl,
-    headline: 'This PR‘s review app has been redeployed!',
+    headline: 'This PR’s review app has been redeployed!',
```

Comment:
This dist has a lot of stuff that isn't in the PR itself, is it possible that it's out-of-date?